spec/PeriodicBackgroundSync-index.bs

```diff
@@ -92,3 +94,60 @@ Fetch requests within the <code>periodicsync</code> event while [=in the backgro
 
 ## History Leaking ## {#history-leaking}
 Fetch requests within the <code>periodicsync</code> event while [=in the background=] may reveal something about the client's navigation history to middleboxes on networks different from the one used to create the [=periodicsync registration=]. For instance, the client might visit site https://example.com, which registers a <code>periodicsync</code> event, but based on the implementation, might not fire until after the user has navigated away from the page and changed networks. Middleboxes on the new network may see the fetch requests that the <code>periodicsync</code> event makes. The fetch requests are HTTPS so the request contents will not be leaked but the domain may be (via DNS lookups and IP address of the request). To prevent this leakage of browsing history, the user agent MAY choose to only fire <code>periodicsync</code> events on the network the [=periodicsync registration=] was made on, with the understanding that it will reduce usability by not allowing synchronization opportunistically.
+
+API Description {#api-description}
+===================================
+## Extensions to the {{ServiceWorkerGlobalScope}} interface ## {#serviceworkerglobalscope-extension}
```

Comment:
Why do we need the custom link IDs here? Dito on line :107, where spacing is inconsistent too.

Reply:
Do you mean the link id #serviceworkerglobalscope-extension? I have added them to each heading, as I've seen other specs do this too. Should they only be specified for links that're referred to within the spec?

Reply:
I've fixed the spacing on :107.

Reply:
If there's a benefit, then yeah, sure, but "API Description" will get the anchor "api-description" automatically already, so it's redundant. For other headers, if they have to be referred to from _outside of the spec_ then it makes sense to add an explicit anchor, but in most cases such references would be to algorithms or definitions.

Reply:
It's hard to know which would need to be referred to from outside of the spec in the future. I'll remove this one for now.

Reply:
Exactly, so best to add them when a need occurs :).

Reply:
I get these warnings if I don't add these:

WARNING: You should manually provide IDs for your headings:
  <h3 line-number="130" class="heading" data-level="6.1" id="extensions-to-the-serviceworkerglobalscope-interface"><span class="content">Extensions to the <code class="idl" nohighlight="" line-number="~130"><a bs-autolink-syntax="{{ServiceWorkerGlobalScope}}" data-link-type="idl" data-lt="ServiceWorkerGlobalScope">ServiceWorkerGlobalScope</a></code> interface</span></h3>
  <h3 line-number="137" class="heading" data-level="6.2" id="extensions-to-the-serviceworkerregistration-interface"><span class="content">Extensions to the <code class="idl" nohighlight="" line-number="~137"><a bs-autolink-syntax="{{ServiceWorkerRegistration}}" data-link-type="idl" data-lt="ServiceWorkerRegistration">ServiceWorkerRegistration</a></code> interface</span></h3>

Reply:
Fair enough then, thanks for checking!